Sprint-1/fix/median.test.js

```diff
@@ -20,7 +21,6 @@ describe("calculateMedian", () => {
   test("doesn't modify the input", () => {
     const list = [1, 2, 3];
     calculateMedian(list);
-
     expect(list).toEqual([1, 2, 3]);
   });
-});
+});
```

Comment:
#### Todo
Can you run this test script to check if the function in `median.js` is implemented correctly, and then update the code in `median.js` accordingly?